i18n/locales/fa-IR/hourofcode/promote/previous-posters.md

```diff
@@ -1,17 +1,17 @@
 ---
-title: Past Hour of Code Posters
+title: پوستر های ساعت کدنویسی های قبلی
 ---
 
 {{ signup_button }}
 
-# Past Hour of Code Posters
+# پوستر های ساعت کدنویسی های قبلی
 
-### Find our posters from previous years to print and hang in your classroom! Looking for the newest posters? [Click here]({{ promote/promote_posters_url }}).
+### پوستر های ما را از از سال های قبل پیدا کنید و آنها را پرینت کنید و به دیوار کلاس بچسبانید! به دنبال جدیدترین پوسترها می گردید؟ [اینجا کلیک کنید.]({{ promote/promote_posters_url }})
```

Comment:
@castro-jorge looks like we've got several issues with this language. This line breaks markdown syntax and https://github.com/code-dot-org/code-dot-org/pull/34189/commits/f1cbdd86fff54725ac74a46fb476fd91d4a698a0#diff-a7ee9f65e863fa36628828b6825bf405R25 and https://github.com/code-dot-org/code-dot-org/pull/34189/commits/f1cbdd86fff54725ac74a46fb476fd91d4a698a0#diff-a7ee9f65e863fa36628828b6825bf405R17 are both removing links